packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -110,14 +111,63 @@ class PopupMenuDivider extends PopupMenuEntry<Never> {
   /// Creates a horizontal divider for a popup menu.
   ///
   /// By default, the divider has a height of 16 logical pixels.
-  const PopupMenuDivider({super.key, this.height = _kMenuDividerHeight});
+  const PopupMenuDivider({
+    super.key,
+    this.height = _kMenuDividerHeight,
+    this.thickness,
+    this.indent,
+    this.endIndent,
+    this.radius,
+    this.color,
+  });
 
   /// The height of the divider entry.
   ///
   /// Defaults to 16 pixels.
   @override
   final double height;
 
+  /// The thickness of the line drawn within the divider.
+  ///
+  /// A divider with a [thickness] of 0.0 is always drawn as a line with a
+  /// height of exactly one device pixel.
+  ///
+  /// If this is null, then the [DividerThemeData.thickness] is used. If
+  /// that is also null, then this defaults to 0.0.
```

Comment:
If these comments are identical to the docs of `Divider`, we can create `{@macro ...}` for the `Divider`'s properties and use `{@template}` for the properties here.